disaster-recovery-webapp/src/main/scala/uk/gov/nationalarchives/webapp/FrontEndServer.scala

```diff
@@ -0,0 +1,29 @@
+package uk.gov.nationalarchives.webapp
+
+import cats.effect.Async
+import cats.syntax.all.*
+import com.comcast.ip4s.*
+import org.http4s.ember.server.EmberServerBuilder
+import org.http4s.implicits.*
+import org.http4s.server.middleware.Logger
+import org.http4s.server.staticcontent.resourceServiceBuilder
+
+object FrontEndServer:
+
+  def run[F[_]: Async]: F[Nothing] = {
+    val httpApp = (
+      FrontEndRoutes.ocflRoutes[F] <+> resourceServiceBuilder[F]("/").toRoutes
+    ).orNotFound
+
+    val finalHttpApp = Logger.httpApp(true, true)(httpApp)
+
+    for {
+      _ <-
+        EmberServerBuilder
+          .default[F]
+          .withHost(ipv4"0.0.0.0")
+          .withPort(port"8080")
```

Comment:
Should the port number be in the `.conf` file?

Reply:
I think it's ok. I can't think of a reason we'd ever want to change it. The port is hard coded on the server it runs on so we don't want to be changing it by accident.